pandas/core/window/rolling.py

```diff
@@ -554,7 +563,11 @@ def homogeneous_func(values: np.ndarray):
             if values.size == 0:
                 return values.copy()
 
-            offset = calculate_center_offset(window) if center else 0
+            if func_type in ["fixed", "weighted"]:
```

Comment:
see comment above.

since calculate_center_offset is now a method, you can just have this return the correct value.